pwndbg/__init__.py

```diff
@@ -72,6 +73,12 @@
 except:
     pass
 
+# this is an unconventional workaround to
+# support unicode printing for python2
+# https://github.com/pwndbg/pwndbg/issues/117
+if six.PY2:
+    reload(sys)
+    sys.setdefaultencoding('utf-8')
```

Comment:
I see your concern, but yes that works well :smile:

PS: python2 should honestly die! :see_no_evil: